### PR TITLE
Add 2020.nlpcovid19 acl submission number 18

### PR DESCRIPTION
**Pull request by Timoeller**
As discussed with Karin Verspoor we can add our accepted ACL covid workshop [submission](https://openreview.net/forum?id=JENSKEEzsoU&noteId=TjyVZ690KGJ) to the proceedings.

I hope the PR is correct: 
- I double-checked the crc32 of other submissions - the tool I am using seems correct.
- I get a "404 Not Found" at https://www.aclweb.org/anthology/2020.nlpcovid19-acl.18.pdf I guess you will need to upload the pdf once the xml changed? 

If you need any more assistance, please let me know.

### CODE REVIEW

**Comment by mjpost** [comment]
Paper is in place. Thank you for submitting the PR1

**Review by mjpost — APPROVED** [review]

**Comment by akoehn** [comment]
The changes of this PR are now also in #1104, so we will not need to merge this PR after #1104 is merged.

**Comment by mjpost** [comment]
Merged with 0db971c44e82f2f20744360ffcc94fdf3f4c1f45.

**Pull request closed without merging**
